src/web/app/pages-instructor/instructor-course-enroll-page/instructor-course-enroll-page.component.ts

```diff
@@ -277,16 +260,24 @@ export class InstructorCourseEnrollPageComponent implements OnInit {
   /**
    * Converts returned student list to a suitable format required by Handsontable.
    */
-  studentListDataToHandsontableData(studentsData: StudentAttributes[], handsontableColHeader: any[]): string[][] {
+  studentListDataToHandsontableData(studentsData: Student[], handsontableColHeader: any[]): string[][] {
     const headers: string[] = handsontableColHeader.map(this.unCapitalizeFirstLetter);
-    return studentsData.map((student: StudentAttributes) => (headers.map(
-        (header: string) => (student as any)[header])));
+    return studentsData.map((student: Student) => (headers.map(
```

Comment:
To keep the code simple, I think you can change the `colHeaders` instead.

Reply:
I think that might overly complicate it actually, since it will change the name of the table headers displayed to users, which will need more code to handle (assuming it's not referenced anywhere else). I could give it a shot if you really want though just let me know